internal/datanode/flow_graph_manager.go

```diff
@@ -33,42 +33,69 @@ import (
 	"github.com/milvus-io/milvus/pkg/util/typeutil"
 )
 
-type flowgraphManager struct {
+type FlowgraphManager interface {
+	Start()
+	Stop()
+
+	// control the total memory water level by sync data.
+	controlMemWaterLevel(totalMemory uint64)
```

Comment:
shall we keep this private method in interface?

Reply:
True, forget to remove this